plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -153,11 +157,13 @@ func (c *PodSecurityPolicyPlugin) Validate(a admission.Attributes) error {
 	pod := a.GetObject().(*api.Pod)
 
 	// compute the context. Mutation is not allowed. ValidatedPSPAnnotation is used as a hint to gain same speed-up.
-	allowedPod, _, validationErrs, err := c.computeSecurityContext(a, pod, false, pod.ObjectMeta.Annotations[psputil.ValidatedPSPAnnotation])
+	allowedPod, pspName, validationErrs, err := c.computeSecurityContext(a, pod, false, pod.ObjectMeta.Annotations[psputil.ValidatedPSPAnnotation])
 	if err != nil {
 		return admission.NewForbidden(a, err)
 	}
 	if apiequality.Semantic.DeepEqual(pod, allowedPod) {
+		ae := a.GetAuditEvent()
+		audit.LogAnnotations(ae, "podsecuritypolicy/validate", pspName)
```

Comment:
we need better keys here, maybe `podsecuritypolicy.admission.k8s.io/policy`

Reply:
👍 we should always use full names